test/react-native/scripts/build-ios.sh

```diff
@@ -0,0 +1,14 @@
+#!
```

Comment:
I'd always do `set -e` so that any errors cause this script to exit with an error.

I'd also be more explicit in the shebang so that this script behaves the same wherever it is run `#!/bin/bash` or `#!/bin/env bash`